### ISSUE TITLE
x/vulndb: potential Go vuln in github.com/gin-gonic/gin: CVE-2023-29401

### ISSUE THREAD

**Issue by motoyasu-saburi** [golang/vulndb#1737]
### Description

Gin is a web framework written in Go.
Gin prior to version v1.9.0 and below is vulnerable to Reflect File Download.
This problem occurs when `FileAttachment()` is used.
A PullRequest to correct this problem has been provided but has not yet been fixed.

### Affected Modules, Packages, Versions and Symbols

```shell
Module: github.com/gin-gonic/gin
Package: github.com/gin-gonic/gin
Versions:
  - Introduced: 1.9.0
Symbols:
  - FileAttachment
```

### Does this vulnerability already have an associated CVE ID?

No

### Pull Request

https://github.com/gin-gonic/gin/pull/3556

### References

Report: 
https://github.com/gin-gonic/gin/issues/3555

**Comment by zpavlinovic** [comment]
@motoyasu-saburi Would you mind updating notifying us here if and when the fix becomes available?

**Comment by zpavlinovic** [comment]
The Go Vulnerability Database has designated this GO-2023-1737 (https://pkg.go.dev/vuln/GO-2023-1737) and CVE-2023-29401. To add a fixed version or otherwise update this report, you can reopen and comment on https://github.com/golang/vulndb/issues/1737.

**Comment by motoyasu-saburi** [comment]
@zpavlinovic Okay, Thank you for your support.

**Comment by herrberk** [comment]
@zpavlinovic a fix for this vulnerability is now available via [v1.9.1](https://github.com/gin-gonic/gin/releases/tag/v1.9.1). Special thanks to @motoyasu-saburi @thinkerou
